department-of-veterans-affairs/content-build
# Improvements to local development experience

Added new script to allow local viewing of vets-website generated assets through a symlink inside the content-build repo.

Testing Instructions:
1) remove `./build`
2) run `yarn build` or `yarn watch` (if you run build you'll need to start a server in ./build/localhost/)
3) Visit the site and notice there are no styles
5) Refresh the site, it should be styled now, as the symlink is created

Running `yarn watch:content` should spin up a site at localhost:3002; if you roll your own, it just can't be on 3001, because vets-website uses it.

Added a flag to not process the symlink just in case it ever gets in the way: `yarn build --noSymlink`
